peripherals/devmock/README.md

```
@@ -0,0 +1,80 @@
+```bash
+curl -X POST http://localhost:8000/__admin/scenarios/reset
+```
+
+## The watchdog re-delivery loop
+
+`mappings/system/reschedule_watchdog.json` scopes a WireMock **webhook** to the
+watchdog task id (`TSK-0014-6018-1456-2416`). When the SDK reschedules that task
+(the `watchdog-defer` handler always raises `DeferError`), the webhook re-POSTs
+the `watchdog-defer` event to the extension after a 5s delay, closing the loop:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)